yumi_moveit_config/config/yumi.srdf

```diff
@@ -24,26 +24,25 @@
     <!---------- Movement Groups ---------->
     <!--- Programmer: Frederick Wachter --->
     <!---- First Modified: 2016-05-20 ----->
-    <!----- Last Modified: 2016-05-20 ----->
     <!--=================================-->
 
-    <!--<group name="arms_and_hands">
-      <group name="left_arm_hand"/>
-      <group name="right_arm_hand"/>
-    </group>-->
+    <group name="full_arms">
+      <group name="left_arm"/>
+      <group name="full_right_arm"/>
```

Comment:
Since the camera is on the left arm. For the feature/TwoGrippers branch, there will be both `full_left_arm` and `full_right_arm`.